products/clamav.md

```diff
@@ -0,0 +1,88 @@
+---
+title: ClamAV
+category: app
+iconSlug: NA
+permalink: /clamav
+releasePolicyLink: https://docs.clamav.net/faq/faq-eol.html
+changelogTemplate: "https://github.com/Cisco-Talos/clamav/releases/tag/clamav-__LATEST__"
+releaseColumn: true
+releaseDateColumn: true
+
+auto:
+-   git: https://github.com/Cisco-Talos/clamav.git
+    regex: '^clamav-(?<major>[0-9]+)\.(?<minor>[0-9]+)(\.(?<patch>[0-9]+))?$'
+
+releases:
+-   releaseCycle: "1.0"
+    releaseDate: 2022-11-28
+    eol: 2025-11-28
+    lts: true
+    latest: "1.0.0"
+    latestReleaseDate: 2022-11-28
+
+-   releaseCycle: "0.105"
+    releaseDate: 2022-05-04
+    eol: 2023-03-28
```

Comment:
I hesitated but I used the worst case scenario to obtain a date. Because according to https://docs.clamav.net/faq/faq-eol.html#regular-non-lts-feature-releases :

> Non-LTS feature releases will be supported with critical patch versions **for at least four (4) months from the initial publication date** of the next feature release or until the next-next feature release is published.

But if you think this is wrong let's change the value to `false`.

Reply:
Done.

Reply:
Netlify checks were blocked again. Amending latest commit to unblock it.

Reply:
Looks good now -> resolved